Remotion/Web/Core/UI/Controls/WebTreeView.cs

```diff
@@ -585,11 +593,18 @@ protected override void Render (HtmlTextWriter writer)
     /// <summary> Overrides the parent control's <c>RenderContents</c> method. </summary>
     protected override void RenderContents (HtmlTextWriter writer)
     {
+      var escapedUpdateMessageTextOrNull = _clickedNodeWithoutChildrenText switch
+      {
+          null => "null",
+          _ => "'" + ScriptUtility.EscapeClientScript(_clickedNodeWithoutChildrenText + " " + GetResourceManager().GetString(ResourceIdentifier.ScreenReaderNoItemFoundText)) + "'"
+      };
```

Comment:
code design : in null-or-else situations, my go to would be ?: instead of a switch with pattern matching. 

It's okay like this, it just feels like overkill because it has five lines instead of three (or one line if no linewrap is needed)

Reply:
I think ?: is horrible readability wise in cases where we have one option as long as the second one, splitting the ? : always feels off IMO